src/pybind/mgr/prometheus/module.py

```diff
@@ -593,7 +593,8 @@ def get_mgr_status(self):
             host_version = servers.get((mgr, 'mgr'), ('', ''))
             if mgr == active:
                 _state = 1
-                ceph_release = host_version[1].split()[-2] # e.g. nautilus
+                if host_version[1]:
+                    ceph_release = host_version[1].split()[-2] # e.g. nautilus
```

Comment:
I think you can just use `self.release_name` instead. See [1] as an example.

[1] https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/controllers/mgr_modules.py#L49

Reply:
I did some testing on this and it is indeed available all the way down to nautilus and it is a lot more stable than getting the version from the servers dict -- it is available even before the servers dict versions are populated.

I have re-pushed the commit with the change.